More/Delegation/JuniorDeveloper.php

```diff
@@ -8,4 +8,11 @@ public function writeBadCode(): string
     {
         return 'Some junior developer generated code...';
     }
+    /**
```

Comment:
Since you are changing the file please work on proper formatting. Thanks.

Reply:
Sure, tnx. Comments was added just to explain reason for change.

Reply:
@carousel - I mean new line before comments of course :)